src/bacnet/cov.c

```diff
@@ -654,9 +652,7 @@ int cov_subscribe_property_decode_service_request(
             data->cancellationRequest = false;
             len += decode_tag_number_and_value(
                 &apdu[len], &tag_number, &len_value);
-            data->issueConfirmedNotifications =
-                decode_context_boolean(&apdu[len]);
-            len++;
```

Comment:
I am not really sure why this is being incremented here.  Wouldn't len be advanced past this option by decode_tag_number_and_value?

Reply:
I agree, len shouldn't be incremented again. Seems like a copy/pasta error.